packages/migrate/src/commands/MigrateDiff.ts

```diff
@@ -147,6 +147,8 @@ ${chalk.bold('Examples')}
       throw new MigrateDiffNeedsPreviewFeatureFlagError()
     }
 
+    loadEnvFile(args['--schema'], false)
```

Comment:
there is no `--schema` argument to diff, is this line right?

Reply:
Indeed! A bit surprised the type safety didn't help...